docs/DeveloperGuide.md

```diff
@@ -177,6 +177,60 @@ Classes used by multiple components are in the `seedu.guest.commons` package.
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Add feature
+
+#### Implementation:
+
+* The `add` command takes in 6 compulsory fields (`Name`, `Phone`, `Email`, `Room Number`, `Date Range` and `Number of Guests`) and 1 optional field (`Request`) and is supported by the `AddCommandParser` that extracts out each of the fields from their respective prefixes.
```

Comment:
Should we name it Room instead of Room Number, based on what we define for the class?

Reply:
Agreed. Changed.